install/first_halyard_boot.sh

```diff
@@ -93,6 +93,21 @@ function replace_startup_script() {
   clear_instance_metadata "startup-script"
 }
 
+function enable_apis() {
```

Comment:
Scopes are handled in the DM jinja file, depending on which providers are enabled. They are in alignment with the APIs that are enabled in this function.